src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -439,15 +440,25 @@ std::string parseCustomJson
 static std::string parseCustomPayload
 (
   ConnectionInfo*             ciP,
+  ngsiv2::CustomPayloadType*  payloadType,
   std::string*                payload,
-  orion::CompoundValueNode**  json,
   bool*                       includePayload,
+  orion::CompoundValueNode**  json,
+  Entity*                     ngsi,
   const Value&                holder
 )
 {
-  if ((holder.HasMember("payload")) && (holder.HasMember("json")))
+  // Text is the one by default
+  *payloadType = ngsiv2::CustomPayloadType::Text;
+
+  unsigned int n = 0;
+  if (holder.HasMember("payload")) n++;
+  if (holder.HasMember("json"))    n++;
+  if (holder.HasMember("ngsi"))    n++;
+
+  if (n > 1)
   {
-    return badInput(ciP, "payload and json fields cannot be used at the same time in httpCustom or mqttCustom");
+    return badInput(ciP, "no more than one payload, json or ngsi field at the same time in httpCustom or mqttCustom");
```

Comment:
This message was not 100% clear when reviewing in the tests. IMHO can be improved

```suggestion
    return badInput(ciP, "only one of payload, json or ngsi fields accepted at the same time in httpCustom or mqttCustom");

Reply:
Sounds better :)

Fixed in 0a8fe56e5